site/en/blog/private-network-access-preflight/index.md

```diff
@@ -67,7 +68,7 @@ the change and adjust accordingly.
     seconds.
     {% endAside %}
 
-2. In Chrome 107 at the earliest:
+2. In ~~Chrome 107 at the earliest~~ Chrome 109:
     * This will begin _only_ if and when compatibility data indicates that the
```

Comment:
We have analyzed compatibility data, so this conditional sentence is outdated. We should update it.

Reply:
Done